### PR TITLE
Update magic auth docs

### PR DESCRIPTION
**Pull request by BrianKariukiDev**

### CODE REVIEW

File: docs/mint.json [hunk 1]
```diff
@@ -90,6 +90,32 @@
...
             ]
+        },
+        {
+            "group": "Session tokens",
+            "pages": [
+                "user-management/session-tokens/jwks",
+                "user-management/session-tokens/access-token",
+                "user-management/session-tokens/refresh-token"
+            ]
+        },
```
Comment (njoguamos) [inline docs/mint.json:101]: You already pushed this code in the previous PR. 

Pushing code twice may lead to conflicts and inconsistent code. 

Remove them,

**Review by njoguamos — CHANGES_REQUESTED** [review]

File: docs/mint.json (hunk 1 above)
Comment (BrianKariukiDev) [reply docs/mint.json:101]: So a pull request should not include the code changes in the previous pull request if not already merged?

Comment (njoguamos) [reply docs/mint.json:101]: Yes.

Comment (BrianKariukiDev) [reply docs/mint.json:101]: Should I also remove the session-tokens folder?

Comment (njoguamos) [reply docs/mint.json:101]: Yes.

**Comment by BrianKariukiDev** [comment]
I have made the changes.

**Review by njoguamos — APPROVED** [review]

**Pull request merged**